src/garrysmod/gamemodes/zombiesurvival/gamemode/zombieclasses/zombie.lua

```diff
@@ -283,4 +283,12 @@ end
 
 if CLIENT then
 	CLASS.Icon = "zombiesurvival/killicons/zombie"
```

Comment:
```suggestion
	CLASS.Icon = "zombiesurvival/killicons/fresh_dead"
```

so that

![image](https://user-images.githubusercontent.com/1774125/93417917-1c6e2a80-f877-11ea-9f6d-60f5de1705fe.png)